pynsee/macrodata/_dwn_idbank_files.py

```diff
@@ -60,19 +61,13 @@ def _dwn_idbank_files():
                 pynsee_idbank_loop_url = False
         except:
             pass
-    
-    session = requests.Session()
```

Comment:
I don't think this should have been removed, wouldn't we get a penalty by removing it?
Anyway, this  should not be part of the PR, please revert

Reply:
Yes, I think this has it's source in @lgeo3's branch originating before https://github.com/InseeFrLab/pynsee/pull/125